test/connectors/PropertyLinkConnectorSpec.scala

```diff
@@ -103,6 +108,19 @@ class PropertyLinkConnectorSpec extends VoaPropertyLinkingSpec {
     whenReady(connector.getMyOrganisationAgents())(_ mustBe organisationsAgentsList)
   }
 
+  "getMyAgentPropertyLinks" must "return agent property links" in new Setup {
+    val emptySearchParams = GetPropertyLinksParameters()
+    val paginationParams = PaginationParams(1, 1, true)
```

Comment:
PaginationParams(1, 15, true) would be better

Reply:
actually PaginationParams() defaults to good values doesn't it?

Reply:
DefaultPaginationParameters

Reply:
Updated to use DefaultPaginationParams